ginga/qtw/Widgets.py

```diff
@@ -1792,6 +1827,11 @@ def start_drag(self):
             self._drag.start(QtCore.Qt.MoveAction)
 
 
+class WidgetMoveEvent(object):
+    def __init__(self, src_widget, child):
+        raise NotImplementedError
+
```

Comment:
Under GTK and GTK3. I didn't realize you also have `pgw` widgets. Perhaps I should also add `__all__` in that. Since you do `import *` from them, I think `__all__` will help that import to be cleaner.